templates/magazine/magazine.css

```diff
@@ -195,12 +195,16 @@ div.section.eloqua-form-container {
   background-color: #D8D8D8;
   position: relative;
   clear: right;
-  width: 756px;
+  width: 100%;
   padding: 35px 15px;
   margin: 48px auto;
+
+  @media (min-width: 774px) {
+    width: 756px;
```

Comment:
Now on the desktop the section driving progress is shrink
<img width="1980" alt="obraz" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/125962117/52f09a6d-3c3b-4595-b119-2ddcc04ba6e9">

Reply:
The media query block is now removed and now it stretches full width.